settings/sample-memcache.settings.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Template for memcache configuration and SASL authentication for Acquia Cloud
+ */
+
+if ($is_ah_env) {
+  $settings['cache']['default'] = 'cache.backend.memcache';
+  $settings['memcache']['sasl'] = [
+  'username' => 'user',
```

Comment:
Is SASL supported within Acquia? I'm wondering if SASL should be included only with a configuration parameter. Additionally, would we want to also include memcache on local environments (or even CI)?